controllers/volumereplicationgroup_controller.go

```diff
@@ -250,6 +270,33 @@ func (v *VolumeReplicationGroupReconciler) HandlePersistentVolumes(
 	return nil
 }
 
+func (v *VolumeReplicationGroupReconciler) CreateVolumeReplicationCRsFromPVC(
+	ctx context.Context,
+	pvc corev1.PersistentVolumeClaim) error {
+	cr := &ramendrv1alpha1.VolumeReplication{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      pvc.Name,
+			Namespace: pvc.ObjectMeta.Namespace,
+		},
+		Spec: ramendrv1alpha1.VolumeReplicationSpec{
+			DataSource: &corev1.TypedLocalObjectReference{
+				Kind: "PersistentVolumeClaim",
+				Name: pvc.Name,
+			},
+			State: "Primary",
+		},
+	}
+
+	log.Info("Created CR: ", cr)
```

Comment:
(nit) "creating" and not "created"? Also, we may want to move this to `log.Debug` to reduce log noise?